go.mod

```
@@ -19,16 +19,16 @@ require (
 	github.com/spf13/viper v1.12.0
 	github.com/stretchr/testify v1.8.0
 	github.com/uber/jaeger-client-go v2.30.0+incompatible
-	go.opentelemetry.io/otel v0.20.0
-	go.opentelemetry.io/otel/exporters/metric/prometheus v0.20.0
-	go.opentelemetry.io/otel/exporters/otlp v0.20.0
-	go.opentelemetry.io/otel/exporters/trace/jaeger v0.20.0
-	go.opentelemetry.io/otel/metric v0.20.0
-	go.opentelemetry.io/otel/oteltest v0.20.0
-	go.opentelemetry.io/otel/sdk v0.20.0
-	go.opentelemetry.io/otel/sdk/export/metric v0.20.0
-	go.opentelemetry.io/otel/sdk/metric v0.20.0
-	go.opentelemetry.io/otel/trace v0.20.0
+	go.opentelemetry.io/otel v1.9.0
+	go.opentelemetry.io/otel/exporters/metric/prometheus v1.9.0
+	go.opentelemetry.io/otel/exporters/otlp v1.9.0
+	go.opentelemetry.io/otel/exporters/trace/jaeger v1.9.0
```

I assume this changed. (https://github.com/frzifus/jaeger-otel-test/commit/96a352b4a181ff43213c7ffc578b39ca1cfe2451)
```suggestion
	go.opentelemetry.io/otel/exporters/jaeger v1.9.0
```